push-docker-manifest/README.md

```diff
@@ -0,0 +1,19 @@
+# push-docker-manifest
+
+
+
+## Usage
+
+```
+jobs:
+  push_manifest:
+    runs-on: ubuntu-latest
+    steps:
+      ...
+      - uses: actions/checkout@v2
```

Comment:
Does this actually require checking out code?

Reply:
No doesn't - I'll remove that step